patches/html-render-webpack-plugin@3.0.1.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/dist/evaluateFromFileSystem.js b/dist/evaluateFromFileSystem.js
+index 8adee8897407a9d496a31f315bd1341e28e0ab8a..6de8dd8a5b72700f55004ba610f36e9a0d7dc638 100644
+--- a/dist/evaluateFromFileSystem.js
++++ b/dist/evaluateFromFileSystem.js
+@@ -33,7 +33,10 @@ function createLinker(parentModulePath, fsModule, rootDir, extraGlobals) {
+         const absPath = path_1.default.join(path_1.default.dirname(parentModulePath), specifier);
+         if (!getFromSourceModules(specifier, fsModule, rootDir)) {
+             logging_1.log(`Using external require for ${specifier} from ${parentModulePath}`);
+-            return require(specifier);
++
++            const absPath = require.resolve(specifier, {paths: [rootDir]})
++
++            return require(absPath);
```

Comment:
This seems to be a bug when using pnpm with html-render-webpack-plugin. The fix will likely be rolled through soon, patching for now.